R/class_country.R

```diff
@@ -76,7 +79,8 @@ daedalus_country <- function(
     contact_matrix = NULL,
     contacts_workplace = NULL,
     contacts_consumer_worker = NULL
-  )
+  ),
+  group_working_age = i_WORKING_AGE
```

Comment:
below you set the default assuming `NULL`, and `i_WORKING_AGE` is not exported so should probably not be visible here in the arg list

```suggestion
  group_working_age = NULL
```

(perhaps update the docs too)